content/en/docs/ambient/overview/APIs/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Ambient APIs
+description: Understanding ambient APIs and its status.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+This page lists all the user facing APIs relevant to the ambient mode and its feature status.
+
+## Labels
+
+| Name | Status |
+| ------------------------------- | -------------------------- |
+| `istio.io/dataplane-mode` | Beta |
+| `istio.io/use-waypoint` | Beta |
+| `istio.io/use-waypoint-namespace` | Beta |
+| `istio.io/waypoint-for` | Beta |
+
+TODO: add link to labels page for each when 3307 is merged.
+
+## Annotations
+
+Below are annotations that can be used to customize your waypoint proxies:
+
+| Name | Status |
+| ------------------------------- | -------------------------- |
+| `gateway.istio.io/service-account` | Alpha |
+| `ambient.istio.io/waypoint-inbound-binding` | Alpha |
```

Comment:
I actually have no idea what this does

Reply:
It's undocumented/WIP/experimental AFAIK: https://github.com/istio/istio/pull/50235

I'd be in favor of simply not mentioning it in this doc at all at this time.

Reply:
Oh yeah I'd leave this completely undocumented; I don't even know if I see sandwich on istio.io anywhere

Reply:
Sounds good, I've removed annotations out.